.github/workflows/cleanup.yml

```diff
@@ -68,3 +74,46 @@ jobs:
             echo "Deleting ${NAME}..."
             gcloud run revisions delete ${NAME} --project="$INTEGRATION_PROJECT_ID" --region="$INTEGRATION_REGION" --quiet --async
           done
+  key_cleanup:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: 'actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab' # ratchet:actions/checkout@v3
+      - uses: 'google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033' # ratchet:google-github-actions/auth@v1
+        with:
+          workload_identity_provider: '${{ env.WIF_PROVIDER }}'
+          service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
+      - uses: 'google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b' # ratchet:google-github-actions/setup-gcloud@v1
+      - name: 'create time span env var'
+        if: '${{ github.event_name == ''workflow_dispatch''}}'
+        shell: 'bash'
+        run: |-
+          echo "TIME_SPAN=${{ inputs.time_span }}" >> $GITHUB_ENV
+      - name: 'create time span env var'
+        if: '${{ github.event_name != ''workflow_dispatch''}}'
+        shell: 'bash'
+        run: |-
+          echo "TIME_SPAN=1d" >> $GITHUB_ENV
```

Comment:
Nit: can we combine it with below, and use it in `Destroy all key version in the keyring` directly?
```
if [ $A = "B" ]; then
>   export a=b
> else
>   export a=b
> fi
```